pkg/client/s3client.go

```diff
@@ -46,6 +47,32 @@ const (
 	AuthEndpoint = "https://iam.cloud.ibm.com/identity/token"
 )
 
+//Func NewS3Client accepts apikey, accesskey, secretkey of the bucket and return the s3 client
+//to perform different s3 operations like upload, delete etc.,
+func NewS3Clientwithkeys(c *Client, accesskey, secretkey, region string) (s3client *S3Client, err error) {
+
+	s3client = &S3Client{}
+	if pkg.Options.APIKey == "" {
+		s3client.ApiKey = os.Getenv("IBMCLOUD_API_KEY")
+	} else {
+		s3client.ApiKey = pkg.Options.APIKey
+	}
```

Comment:
If access key and secret passed, do we still need API key?

Reply:
No, its not needed. Bluemix client is also not needed. Will remove both from this function